### ISSUE TITLE
When browsing the accessible buffer with ctrl+up or ctrl+down nothing is read by orca

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#182099]
Type: <b>Bug</b>
cc: @meganrogge  @isidorn 

I open a terminal in VSCode, execute some commands and press shift+tab to move the focus to the accessible buffer.

Once this is done I use ctrl+up and ctrl+down to navigate between the commands but even though the focus is moved nothing is read by orca.

It would be important, at least in my opinion, that each time ctrl+up or ctrl+down is pressed at least one line of the command is read.

VS Code version: Code - Insiders 1.79.0-insider (efb49cc271d6eea5634ac395e36e1e4cd108a447, 2023-05-10T05:23:17.106Z)
OS version: Linux x64 6.2.14-300.fc38.x86_64
Modes:
Sandboxed: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz (8 x 2500)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|1, 2, 3|
|Memory (System)|15.26GB (7.13GB free)|
|Process Argv|-n --crash-reporter-id bc02adbd-6c3a-44b8-af92-55103a608af7|
|Screen Reader|yes|
|VM|0%|
|DESKTOP_SESSION|gnome-xorg|
|XDG_CURRENT_DESKTOP|GNOME|
|XDG_SESSION_DESKTOP|gnome-xorg|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (44)</summary>

Extension|Author (truncated)|Version
---|---|---
doxdocgen|csc|1.4.0
vscode-markdownlint|Dav|0.50.0
vscode-eslint|dba|2.4.0
EditorConfig|Edi|0.16.4
copilot|Git|1.86.82
rest-client|hum|0.25.1
better-cpp-syntax|jef|1.17.2
docomment|k--|1.0.0
azure-pipelines|ms-|1.208.0
vscode-docker|ms-|1.25.0
csharp|ms-|1.25.7
vscode-dotnet-runtime|ms-|1.6.0
vscode-kubernetes-tools|ms-|1.3.11
remote-containers|ms-|0.293.0
remote-ssh|ms-|0.101.2023050515
remote-ssh-edit|ms-|0.84.0
azure-account|ms-|0.11.4
azurecli|ms-|0.5.0
cmake-tools|ms-|1.14.31
cpptools|ms-|1.15.4
cpptools-extension-pack|ms-|1.3.0
makefile-tools|ms-|0.7.0
powershell|ms-|2023.5.0
remote-explorer|ms-|0.5.2023040509
vsliveshare|ms-|1.0.5864
oracledevtools|Ora|21.5.0
fabric8-analytics|red|0.3.6
java|red|1.19.2023050904
vscode-xml|red|0.25.2023050404
vscode-yaml|red|1.12.2
sonarlint-vscode|Son|3.17.0
cmake|twx|0.0.17
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
vscode-boot-dev-pack|vmw|0.2.1
vscode-spring-boot|vmw|1.46.0
vscode-java-debug|vsc|0.50.2023042406
vscode-java-dependency|vsc|0.22.2023042900
vscode-java-pack|vsc|0.25.2023042006
vscode-java-test|vsc|0.38.2023042703
vscode-maven|vsc|0.41.2023042103
vscode-spring-boot-dashboard|vsc|0.13.0
vscode-spring-initializr|vsc|0.11.2
gitblame|wad|10.2.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256198
vstes627cf:30244335
vslsvsres303:30308271
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
vscod805cf:30301675
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30404738
cppdebug:30492333
vsclangdf:30492506
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30618038
pylantcb52:30590116
pyind779:30611226
pythonsymbol12:30651887
6233i204:30725950
azdwalk:30721579
pythonms35:30671666
cai9j608:30680470
24365598:30687740
pythonfmttext:30716741
pythoncmvfstr:30726892
fixshowwlkth:30724385
hidesbindicator:30724476
pythongtdpath:30726887

```

</details>

**Comment by meganrogge** [comment]
@jvesouza thank you for the issue. are you using a powershell terminal?

**Comment by jvesouza** [comment]
> @jvesouza thank you for the issue. are you using a powershell terminal?
I am using a bash terminal.

**Comment by meganrogge** [comment]
Thanks - this will only work for terminals with supported shell integration. Could you pls run `Terminal: Run recent command` and inspect the quick pick items present there? If there are none, something is going on with shell integration.

**Comment by jvesouza** [comment]
When executing the command "Terminal: Run recent command" the previously executed commands were presented to me.

But I believe I have not explained the problem clearly.

The problem can be reproduced executing the following steps.

1. Launch a terminal.
2.  execute the following commands:
```
echo teste 1
echo teste 2
echo teste 3
```

3. Press shift+tab
4. Press ctrl+up 3 times.

Expected:
Each time ctrl+up was pressed orca should read the command to which the focus was moved.
Actual:
Focus is moved to the command but orca does not read which command the focus was moved to.

After I press ctrl+up 3 times I can navigate normally in the buffer accessible from the command to which the focus was moved.

**Comment by meganrogge** [comment]
Thanks for that info. I believe this is an Orca issue as those steps work for me when using both NVDA and VoiceOver.

**Comment by jvesouza** [comment]
I believe this should be closed because I can navigate the accessible buffer using alt+up arrow and alt+down arrow as expected. Therefore, in my opinion, the problem is resolved.
